src/behaviors/sortable-store.js

```diff
@@ -1,4 +1,100 @@
-export function SortableStore() {
-  return () => {
+import angular from 'angular';
+import {Behavior} from './behavior';
+import {Handler as handlerDecorator} from '../store';
+
+import {
+  Transformer,
+  Transformable as transformableDecorator
+} from './transformable';
+
+import {
+  addBehavior,
+  camelcase,
+  Inject as injectDecorator
+} from '../utils';
+
+export class SortableStoreBehavior extends Behavior {
+  constructor(instance, {collection} = {}) {
+    super(...arguments);
+
+    this.collection = collection || 'items';
+  }
+
+  get $filter() {
+    return this.instance.$filter;
+  }
+
+  get transformableCollection() {
+    return this.instance.transformables[this.collection];
+  }
+
+  get transformer() {
+    if (!this._transformer) {
+      this._transformer = new Transformer('sortableStore', items => items);
+    }
+    return this._transformer;
+  }
+
+  onSortChange(expression = false) {
+    const collection = this.transformableCollection;
+    const transformer = this.transformer;
+
+    if (angular.isString(expression) || angular.isArray(expression)) {
+      transformer.fn = items => {
+        return this.$filter('orderBy')(items, expression);
+      };
+    } else {
```

Comment:
it's not clear, which cases are covered by the else branch. could you please add a comment?